docs/src/reference/compilation.rst

```diff
@@ -184,7 +184,7 @@ be found in the `distutils documentation`_. Some useful options to know about
 are ``include_dirs``, ``libraries``, and ``library_dirs`` which specify where
 to find the ``.h`` and library files when linking to external libraries.
 
-.. _distutils documentation: http://docs.python.org/extending/building.html
+.. _distutils documentation: https://docs.python.org/3/extending/building.html
```

Comment:
The way it's set up is to automatically redirect to the latest version when no version is requested, and that's a good thing. Please leave this as it is.